.test-infra/jenkins/PostcommitJobBuilder.groovy

```diff
@@ -57,8 +60,15 @@ class PostcommitJobBuilder {
   }
 
   void defineAutoPostCommitJob(name) {
+    // default build schedule
+    String buildSchedule = 'H H/6 * * *'
+    try {
+      buildSchedule = scope.getProperty('buildSchedule')
+    } catch (MissingPropertyException ignored) {
+      // do nothing
+    }
```

Comment:
instead of try catch block, can do something like 
```suggestion
    String buildSchedule = scope.hasProperty('buildSchedule')?.getProperty(scope) ?: 'H H/6 * * *'
```

Reply:
I see. I looked again at how buildSchedule is passed into this method, would it not be cleaner to create a new parameter in `PostcommitJobBuilder.postCommitJob()`?

Reply:
Makes sense, thanks for the explanation. Have you considered defining the value inside the closure? So as to be accessed with `jobDefinition.buildSchedule`? I just haven't seen a variable declared like this before in our repo and don't know if there's any downsides to it. 

Up to you, I'll defer to your judgement here.